client/src/styles/legacy/_style.scss

```
@@ -48,6 +48,30 @@ html, body {
   }
 }
 
+.cms-container-skip-link {
```

```suggestion
.skip-link {
```
Seems like it makes sense for it to be quite loosely named? That way developers can safely reuse this CSS class for any skip link they might want to add to the CMS.

If you don't want the JS to apply to other skip links, you could give the element both classes in the markup (one for the JS hook and one for CSS) - but it looks like the JS is written in a way that should work for any custom skip link as well.

---

The styles are really not designed to be reusable e.g. top/left: 3px;

> but it looks like the JS is written in a way that should work for any custom skip link as well.

I hadn't really thought of that, I guess it does, though at the same time do we really need to think about making this reusable? I don't think we're planning on adding any more skip links?

---

> do we really need to think about making this reusable? I don't think we're planning on adding any more skip links?

I had thought it was already reusable - and so changing the name would be an easy way to open it up for reuse (e.g. if an external accessibility review indicates we should use them to skip the gridfield table to get to pagination or something) or if other developers want to add skiplinks for whatever purposes they may have.

But you're right the position is very specific. We can always refactor some of this to be reusable if we find ourselves adding more skiplinks later.